pipenv/project.py

```diff
@@ -470,7 +465,7 @@ def pipfile_location(self):
             try:
                 loc = pipfile.Pipfile.find(max_depth=PIPENV_MAX_DEPTH)
             except RuntimeError:
-                loc = None
+                loc = "Pipfile"
```

Comment:
In case the Pipfile is not found it returns None. Then the following lines will be broken.
```python
@property
    def name(self):
        if self._name is None:
            self._name = self.pipfile_location.split(os.sep)[-2]
        return self._name
```